src/reducer.js

```diff
@@ -2,18 +2,25 @@ import {extend} from "./utils.js";
 import offers from "./mocks/offers";
 import {ActionType} from "./actions";
 
+const isOffersExist = (offersData) => {
+  if (!(offersData instanceof Array)) {
```

Comment:
можно написать и так

```suggestion
  if (!Array.isArray(offersData)) {
```